.github/workflows/release.yml

```diff
@@ -80,6 +80,9 @@ jobs:
           - os-name: windows
             os: windows-2022
             bazel-config: release_windows
+        exclude:
```

Comment:
Nit: This works, but it's easier to remove `ubuntu-22.04-arm` from the os list instead (the `linux-arm64` configuration still gets created manually, with this we're just creating an entry for ubuntu-22.04-arm with X64 only to manually exclude it later)